### PR TITLE
Add indexer configuration

### PR DESCRIPTION
**Pull request by elv-zenia**
Add a view for managing indexer configuration fields.

image

### CODE REVIEW

**Comment by elv-zenia** [comment]
Relates to #46

**Comment by elv-kevin** [comment]
This looks good, we just need a way to determine if the indexer page should be shown, because 99% of the time it shouldn't. 

I've deployed a new asset manager to production (core.v3, core.demov3) that allows you to set this in the type spec configuration. The field is in the metadata of the content type of the object, `public/title_configuration/show_indexer_settings`. The indexer tab should only be shown if that flag is set in the object's type.

**Comment by elv-zenia** [comment]
@elv-kevin Updated

File: src/stores/Object.js [hunk 1]
```diff
@@ -685,6 +788,45 @@ class ObjectStore {
...
+    if(!objectId) objectId = Fabric.utils.DecodeVersionHash(versionHash).objectId;
+
+    return yield Fabric.LatestVersionHash({
+      objectId,
+      versionHash
+    });
+  });
+
+  @action.bound
+  GetIndexerSetting = flow(function * ({typeId}) {
```
Comment (elv-kevin) [inline src/stores/Object.js:820]: this is actually probably already present in `object.typeInfo`. That's where it determines manage app url, which is also in type meta.

**Review by elv-kevin — CHANGES_REQUESTED** [review]

File: src/stores/Object.js (hunk 1 above)
Comment (elv-zenia) [reply src/stores/Object.js:820]: Ah you're right

**Pull request merged**